Detectors/TPC/calibration/SpacePoints/src/TrackResiduals.cxx

```diff
@@ -70,11 +70,24 @@ void TrackResiduals::setY2XBinning(const std::vector<float>& binning)
     LOG(error) << "Binning already initialized, not changing y/x binning";
     return;
   }
-  int nBins = binning.size() - 1;
+
+  if (binning.size() == 0) {
+    LOGP(info, "Empty binning provided, will use default uniform y/x binning with {} bins", mNY2XBins);
+    return;
+  } else if (binning.size() == 1) {
```

Comment:
Fine with me, although in principle for uniform binning one can also call `setNY2XBins` before doing the initialization. But I see that this way will make it simpler to use the script with different binning settings.